README.md

```diff
@@ -155,6 +155,10 @@ on:
   issues:
     types: [opened]
 
+# see: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/controlling-permissions-for-github_token
+permissions:
+  pull-requests: 'write'
```

Comment:
I would expect `issues` `write` to be required for `on: issues` triggered workflows, not `pull-requests`.